Mathlib/Control/LawfulFix.lean

```diff
@@ -225,13 +233,17 @@ variable [(x y : _) → OmegaCompletePartialOrder <| γ x y]
 
 open OmegaCompletePartialOrder.Chain
 
-theorem continuous_curry : Continuous <| monotoneCurry α β γ := fun c ↦ by
-  ext x y
-  dsimp [curry, ωSup]
-  rw [map_comp, map_comp]
-  rfl
+theorem ωScottContinuous_curry :
+    ωScottContinuous (monotoneCurry α β γ) :=
+  ωScottContinuous.of_map_ωSup_of_orderHom fun c ↦ by
+    ext x y
+    dsimp [curry, ωSup]
+    rw [map_comp, map_comp]
+    rfl
 
-theorem continuous_uncurry : Continuous <| monotoneUncurry α β γ := fun c ↦ by
+theorem continuous_uncurry :
+    ωScottContinuous (monotoneUncurry α β γ) :=
+    .of_map_ωSup_of_orderHom fun c ↦ by
```

Comment:
Here is a concrete example
```suggestion
theorem ωScottContinuous_uncurry :
    ωScottContinuous (monotoneUncurry α β γ) :=
    .of_map_ωSup_of_orderHom fun c ↦ by
  ext ⟨x, y⟩
  dsimp [uncurry, ωSup]
  rw [map_comp, map_comp]
  rfl

@[deprecated ωScottContinuous_uncurry  (since := "2024-08-26")]
theorem continuous_uncurry : Continuous <| monotoneUncurry α β γ := fun c ↦ by
```